src/main/java/net/snowflake/ingest/connection/ServiceResponseHandler.java

```diff
@@ -145,7 +177,9 @@ public static HistoryRangeResponse unmarshallHistoryRangeResponse(
     }
 
     // handle the exceptional status code
-    handleExceptionalStatus(response, requestId, ApiName.LOAD_HISTORY_SCAN);
+    response =
```

Comment:
do we use the response anywhere?

Reply:
[Here](https://github.com/snowflakedb/snowflake-ingest-java/blob/6927aaaf0b68cfa6c1e40df28512a535d0f6a661/src/main/java/net/snowflake/ingest/connection/ServiceResponseHandler.java#L184)